backend/app/routers/datasets.py

```diff
@@ -452,8 +407,14 @@ async def delete_dataset(
         await AuthorizationDB.find(
             AuthorizationDB.dataset_id == PydanticObjectId(dataset_id)
         ).delete()
+
+        # don't delete standard license
+        standard_license_ids = [license.id for license in standard_licenses]
```

Comment:
Thank you for catching the bug.

I wonder if we really need this check. the license ids for standard licenses are not in the mongodb. Those will never get deleted. But I guess it never hurts to add a check and never call the delete_license function.

Thanks you for runnning the pre-commit properly on my PR changes.

Reply:
I run the pre-commit run all and fixed more formatting issues. Please check.